src/libmongoc/doc/mongoc_client_set_timeout_ms.rst

```diff
@@ -0,0 +1,32 @@
+:man_page: mongoc_client_set_timeout_ms
+
+mongoc_client_set_timeout_ms()
+==============================
+
+Synopsis
+--------
+
+.. code-block:: c
+
+  bool
+  mongoc_client_set_timeout_ms (mongoc_client_t *client, int64_t *timeout_ms, bson_error_t *error)
```

Comment:
Why is ` int64_t *timeout_ms` a pointer? Here and for the other setters as well.

Reply:
Typo! Fixed.